### ISSUE TITLE
Turtle output should keep isolated blank node on the same line

### ISSUE THREAD

**Issue by jamesaoverton** [knotation/knotation-cljc#11]
See https://github.com/knotation/knotation-examples/blob/master/basic-rdf.md#example-1

This Knotation input
```
: <http://example.com/subject>
<http://example.com/predicate>; <https://knotation.org/datatype/link>: _:blank-node
```
should render to this Turtle
```
<http://example.com/subject>
  <http://example.com/predicate> _:blank-node .
```
but currently the blank node is put on a new line
```
<http://example.com/subject>
  <http://example.com/predicate>
  _:blank-node .
```

I know this seems trivial, but pretty Turtle is important.

**Comment by inaimathi** [comment]
Addressed in https://github.com/IEDB/ONTIE/commit/fae51110fbf78eab7e27306cc47a487b9b5bcd33 and https://github.com/knotation/knotation-cljc/commit/d681393b071d4b95354f16b7d43d6a66cbd0e74b + https://github.com/knotation/knotation-cljc/commit/3c9a6fb18430d467cb5ad8518e1a748dce056b12

**Comment by jamesaoverton** [comment]
Closed by 7fe91961400af4f36d01dc2f25b5782690aa56f2
